core/Cargo.toml

```diff
@@ -156,11 +157,11 @@ num_cpus = "1.13"
 num_enum = "0.7.2"
 octocrab = "0.41"
 once_cell = "1"
-opentelemetry = "0.24.0"
-opentelemetry_sdk = "0.24.0"
-opentelemetry-otlp = "0.17.0"
-opentelemetry-semantic-conventions = "0.16.0"
-opentelemetry-appender-tracing = "0.5"
+opentelemetry = "0.30.0"
+opentelemetry_sdk = "0.30.0"
+opentelemetry-otlp = { version = "0.30.0", default-features = false }
+opentelemetry-semantic-conventions = "0.30.0"
+opentelemetry-appender-tracing = "0.30.0"
```

Comment:
Motivated by `axum` upgrade (these crates use `axum` via `tonic`). I've checked the related changes locally, but not 100% sure they work in all cases. So they can be extracted to a separate PR if desired.